project/Build.scala

```diff
@@ -129,13 +129,14 @@ object DottyBuild extends Build {
     settings(publishing)
 
 
-  lazy val `dotty-interfaces` = project.in(file("interfaces")).
+  lazy val `dotty-interfaces`: Project = project.in(file("interfaces")).
     settings(sourceStructure).
+    dependsOn(`dotty-library`).
     settings(
       // Do not append Scala versions to the generated artifacts
       crossPaths := false,
       // Do not depend on the Scala library
-      autoScalaLibrary := false,
```

Comment:
yes, that's why this commit is called a HACK. 
I propose to discuss on a Dotty meeting what should be the proper way to do it build-wise.